packages/react-components/src/layout/src/Flex.tsx

```diff
@@ -179,9 +142,10 @@ export function InnerFlex({
                     ),
                     style: {
                         flexDirection: direction && `${direction}${reverse ? "-reverse" : ""}`,
-                        alignContent: alignContent,
-                        alignItems: alignItems,
-                        justifyContent: justifyContent,
+                        // Normalize values until Chrome support `start` & `end`, https://developer.mozilla.org/en-US/docs/Web/CSS/align-items.
+                        alignContent: alignContent && alignContent.replace("start", "flex-start").replace("end", "flex-end"),
+                        alignItems: alignItems && alignItems.replace("start", "flex-start").replace("end", "flex-end"),
+                        justifyContent: justifyContent && justifyContent.replace("start", "flex-start").replace("end", "flex-end"),
                         flexWrap: !isNil(wrap) ? "wrap" : undefined,
                         "--o-ui-gap": gap && (isString(gap) ? gap : `var(${Spacing[(gap) - 1]})`)
```

Comment:
At the moment, there is an error on this line. 
```
Type '{ flexDirection: Property.FlexDirection; alignContent: string; alignItems: string; justifyContent: string; flexWrap: "wrap"; "--o-ui-gap": string; }' is not assignable to type 'CSSProperties'.
  Object literal may only specify known properties, and '"--o-ui-gap"' does not exist in type 'CSSProperties'.ts(2322)
```

This is due to the fact that --o-ui-gap is a custom property. After some research, there is multiple ways to handle this scenario : 

**Allow custom property for this object only :** 
Using type assertion, you can allow this property on this object. The downside is that we have to do this synthax everytime we have one of these scenarios : 
```
    ["--o-ui-gap" as any]: gap && (isString(gap) ? gap : `var(${Spacing[(gap) - 1]})`)
```
Source : https://github.com/frenic/csstype#what-should-i-do-when-i-get-type-errors 

**The second option is Augment the CSSProperties to allow this property**
Using [module augmentation](https://www.typescriptlang.org/docs/handbook/declaration-merging.html#merging-interfaces), we can allow this property everywhere

This is painful to write, but we ensure type validation

So at the root of Flex.tsx, we add the following lines : 
```
declare module 'csstype' {
  interface Properties {
    // Add a CSS Custom Property
    '--o-ui-gap'?:  any; // or the right typing.
  }
}
```
Source : https://github.com/frenic/csstype#what-should-i-do-when-i-get-type-errors

**The last solution, wich might be the one we want.** 
We use module augmentation to allow any extra properties.

somewhere in the code, we add this : 
```
declare module 'csstype' {
  interface Properties {
     // ...or allow any other property
    [index: string]: any;
  }
}
```
Allowing any extra parameters to be passed. The cons are that we don't type those properties, but we don't have to add extra synthax in the components